NguyenVanViet2004/FE-GlowUp
# Loading when data canceling booking

## Related issues
Resolve #79 

## Summary

- Fix duplicate id radio button error
- Cancel booking without reloading data
- Update profile information in redux, storage, server
- Fomat price from USD to VND

## How to test
- Cancel booking without reloading data

1.  In the BookingUpcoming screen to cancel the booking
2.  In the BookingCancelled to check

- Update profile information in redux, storage, server

1. Perform profile update on the ProfileUserTemplate screen
2. Return to the ProfileTemplate check screen

## Concern points

## Screenshots
